### DISCUSSION TITLE
Loading failed for the <script> with source “https://samikhan69420.github.io/flutter_bootstrap.js”.

### DISCUSSION THREAD

**Discussion by iamsami101** [community/community#147941] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I made a repo to host my portfolio website and at first, I was using the main branch to host the source code for my site but I made this site in flutter so it didn't show the source code it only showed the compiled javascript, so I made a separate branch for the compiled javascript source code and wanted to push the flutter source code on the main branch, the original link for the website was `https://samikhan69420.github.io` but after adding the new branch and changing the repository name to "`portfolio`", it changed to ﻿`https://samikhan69420.github.io/portfolio` and then it started giving an error which says "`Loading failed for the <script> with source “https://samikhan69420.github.io/flutter_bootstrap.js”.`", notice how it does not include the "`/portfolio`" extention. How do I make it so it changes back to without the "`portfolio`" extention.

**Comment by lemorage** [comment, marked as answer]
You have to rename your repo name back to what it originally was. The reason was that, when you host a website on GitHub Pages under a project repo (i.e., not a repository named username.github.io), GitHub will try to serve the site under a URL that includes the repo name as a subdirectory, `https://samikhan69420.github.io/portfolio` in your case.

> source: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages
